util/exec.go

```diff
@@ -23,6 +23,7 @@ import (
 // from the binary's stdout and stderr to this program's stdout and stderr.
 func Exec(bin string, args ...string) error {
 	cmd := exec.Command(bin, args...)
```

Comment:
The `exec.Command` function does that for me: https://golang.org/src/os/exec/exec.go?s=3999:4044#L112